webapp/channels/src/components/autosize_textarea.tsx

```diff
@@ -168,6 +168,7 @@ const AutosizeTextarea = React.forwardRef<HTMLTextAreaElement, Props>(({
                 {...otherProps}
                 role='textbox'
                 aria-label={placeholderAriaLabel}
+                aria-labelledby={`${id}_label`}
```

Comment:
The goal was to make reader read the label above the textarea. I realised that the `for` should work but it was not working due to `aria-label` present here so the reader was reading placeholder everytime because of ` aria-label={placeholderAriaLabel}`. 
Adding `aria-labelledby` fixed the issue because it overrides `aria-label` but the actual fix could be to just remove the `aria-label` since `for` is present. 
Updated with this change.